src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java

```diff
@@ -270,8 +267,12 @@ protected final void verify(Checker checker,
     }
 
     /**
-     *  We keep two verify methods with separate logic only for convenience of debugging.
-     *  We have minimum amount of multi-file test cases.
+     *  Performs verification of the given files with the given file name.
+     *  Uses provided {@link Checker} instance.
+     *  Expected messages are represented by the array of strings.
+     *  This implementation uses overloaded
+     *  {@link AbstractModuleTestSupport#verify(Checker, File[], Map)}
+     *  method inside.
```

Comment:
I think that we would be better served by slightly improving the `@param` tags and keeping this description to one or two sentences.  Perhaps something like: `Performs verification of expected violations on given files.` or something like that. `This implementation uses overloaded...` seems redundant or unnecessary; I am not sure that anyone needs to know that to use this method.

Reply:
The javadoc of other verify methods has the same structure (and redundancy). Should all be modified?

Reply:
It's up to you. If you change the others, I would make it a separate commit (`doc`).

Reply:
Sure, I'll do it in the next PR.
@romani please verify and open issue.

Reply:
Please remove `This implementation uses  ...`
It is not required, not sure how all other methods get such description.
Javadoc should not explain logic of code if it is easy to read logic

Reply:
done

Reply:
Resolved.